src/current/v24.3/ldap-authentication.md

```diff
@@ -0,0 +1,149 @@
+- `ldapbasedn`: Base DN for user searches
+- `ldapbinddn`: Service account DN for directory searches
+- `ldapbindpasswd`: Service account password
+- `ldapsearchattribute`: Attribute to match against SQL usernames
+- `ldapsearchfilter`: LDAP filter to restrict valid users
+
+For example:
+
+~~~ sql
+SET CLUSTER SETTING server.host_based_authentication.configuration = '
+host    all    all    all    ldap    ldapserver=ldap.example.com 
+    ldapport=636 
+    "ldapbasedn=ou=users,dc=example,dc=com" 
+    "ldapbinddn=cn=readonly,dc=example,dc=com" 
+    ldapbindpasswd=readonly_password 
+    ldapsearchattribute=uid 
+    "ldapsearchfilter=(memberof=cn=cockroachdb_users,ou=groups,dc=example,dc=com)"';
+~~~
+
+If you also intend to [configure LDAP Authorization]({% link v24.3/ldap-authorization.md %}#configuration), you will need to include an additional LDAP parameter, `ldapgrouplistfilter`.
+
+For more information on the HBA fields and HBA in CockroachDB, generally, refer to [SQL Authentication configuration](https://www.cockroachlabs.com/docs/stable/security-reference/authentication#authentication-configuration).
+
+### Step 3: Configure TLS (Optional)
+
+If using LDAPS with a certificate signed by a custom Certificate Authority (CA) that is not in the system's trusted CA store, you'll need to configure the CA certificate. This step is only necessary when using certificates signed by your organization's private CA or other untrusted CA.
+
+Set the custom CA certificate:
+
+~~~ sql
+SET CLUSTER SETTING server.ldap_authentication.domain_ca = '<PEM_ENCODED_CA_CERT>';
+~~~
+
+Configure a client certificate for mTLS if required:
+
+~~~ sql
+SET CLUSTER SETTING server.ldap_authentication.client.tls_certificate = '<PEM_ENCODED_CERT>';
+SET CLUSTER SETTING server.ldap_authentication.client.tls_key = '<PEM_ENCODED_KEY>';
+~~~
+
+### Step 4: Sync Database Users
+
+Before LDAP authentication can be used for a user, the username must be created directly in CockroachDB. You will, therefore, need to establish an automated method for keeping users in sync with the directory server, creating and dropping them as needed.
+
+For Azure AD deployments, the username typically corresponds to the `sAMAccountName` field from AD's person object, which is specified in the HBA configuration using `ldapsearchattribute=sAMAccountName`.
+
+{{site.data.alerts.callout_info}}
+SQL usernames must comply with CockroachDB's [username requirements](https://www.cockroachlabs.com/docs/v24.2/create-user#user-names). Ensure that the values in the LDAP `sAMAccountName` field (or other mapped attribute) meet these requirements.
+{{site.data.alerts.end}}
+
+To create a single user:
+
+~~~ sql
+CREATE ROLE username LOGIN;
+~~~
+
+To create users in bulk:
+
+1. Export usernames from the directory server.
+1. Produce a `.sql` file with a [`CREATE ROLE`](https://www.cockroachlabs.com/docs/v24.2/create-role) statement per user, each on a separate line.
+
+    ~~~ sql
```

Comment:
Maybe needs copy block?

(Same feedback for remaining code blocks that you want users to copy and adapt)